src/index.js

```diff
@@ -67,7 +68,7 @@ module.exports = options => {
       }
 
       generateSlug = async str => {
-        const slug = slugify(str, { lower: true });
+        const slug = slugify(str, Object.assign({}, { lower: true }, opts.slugifyOptions));
```

Comment:
We compile with Babel, so you could just use: 

```js
        const slug = slugify(str, { lower: true, ...opts.slugifyOptions });
```

Reply:
@calvinl Cheers i'll update it